third-party/realdds/src/dds-device-watcher.cpp

```diff
@@ -20,29 +21,32 @@ using namespace realdds;
 
 dds_device_watcher::dds_device_watcher( std::shared_ptr< dds_participant > const & participant )
     : _device_info_topic(
-        new dds_topic_reader( topics::device_info::create_topic( participant, topics::device_info::TOPIC_NAME ) ) )
+        new dds_topic_reader( topics::flexible_msg::create_topic( participant, "realsense/device-info" ) ) )
```

Comment:
Why we don't keep the name at a dedicated header?
It will be hard to follow hard coded names all over the code..

Reply:
I can put `device_info::TOPIC_NAME` in `device-info-msg.h` for this case.
But `notification` and `control` are also hard coded and I don't have a file to put them in...

There are only 2 places for each topic when we use the name. One for the server and one for the client

Reply:
Your choice :)